openedx_events/content_authoring/signals.py

```diff
@@ -276,3 +276,16 @@
         "library_collection": LibraryCollectionData
     }
 )
+
+# .. event_type: org.openedx.content_authoring.course.post.import.v1
+# .. event_name: POST_COURSE_IMPORT
+# .. event_key_field: catalog_info.course_key
+# .. event_description: Fired after a course is imported
+# .. event_data: CourseData
+# .. event_trigger_repository: openedx/edx-platform
+POST_COURSE_IMPORT = OpenEdxPublicSignal(
```

Comment:
Wondering about naming, and whether the event should be named `COURSE_IIMPORT_COMPLETED`? It’s an interesting question, where pre/post is about timing, and started/completed is more about the actual event that took place, and implies the pre/post.

Reply:
@robrap Yes, I was also confused about the naming here but settled with post... Happy to change it.

Reply:
No worries. I do think `COURSE_IMPORT_COMPLETED` would be more consistent with how events have been named to date.

Reply:
@navinkarkera: Everything else looks fine to me, but you may still want a review from someone else who has closer ties to content authoring. Thanks.

Reply:
@robrap Done.